godel/config/dist-plugin.yml

```diff
@@ -1,4 +1,23 @@
 products:
+  resource-reservation-conversion-webhook:
+    build:
+      main-pkg: resource-reservation-conversion-webhook
+      version-var: main.version
+      environment:
+        CGO_ENABLED: "0"
+      os-archs:
+        - os: linux
+          arch: amd64
+    docker:
+      docker-builders:
+        webhook:
+          type: default
+          context-dir: resource-reservation-conversion-webhook/docker
+          tag-templates:
+            release: "{{Repository}}palantirtechnologies/resource-reservation-conversion-webhook:{{Version}}"
```

Comment:
Can we publish a new version only when we've changed that project? I don't think it's a big blocker for now, but I wonder if Godel has proper support for this, and if so, we should use it

Reply:
Let me take a look!

Reply:
From what I can see, it doesn't look like it supports this